content/docs/get-started/index.md

```diff
@@ -165,3 +162,363 @@ requirements:
 ```
 
 </details>
+
+<details>
+
+### Click to learn about model codification
+
+#### Why do it the MLEM way ?
+
+Saving models to files or loading them back into python objects may seem like a
+deceptively simple task at first. For example, `pickle` and `torch` libraries
```

Comment:
This info should be somewhere else I think. In a guide probably. If we feel the need to make long explanations (that end up hidden) here, we probably didn't present this idea properly in the first place.